test/integration/api/datasets.js

```diff
@@ -1,5 +1,11 @@
 const { testService } = require('../setup');
 const testData = require('../../data/xml');
+const config = require('config');
+const { Form } = require('../../../lib/model/frames');
+const { getOrNotFound } = require('../../../lib/util/promise');
+const { omit } = require('ramda');
+
+// TODO merge with test/integration/api/forms/dataset.js
```

Comment:
this sounds good!
in one of my PRs (https://github.com/getodk/central-backend/pull/633) i moved something with a form draft path from dataset into forms/dataset.